src/BloomExe/MiscUI/ToastNotifier.cs

```diff
@@ -157,6 +157,11 @@ private void GoDownTimerTick(object sender, EventArgs e)
 
 		private void ToastNotifier_Click(object sender, EventArgs e)
 		{
+			// Certain scenarios (like clicking to bring up a yellow screen) need these lines else the yellow screen
+			// is closed by a close-this-dialog either already on the stack or about to be added by GoDownTimerTick.
+			_goDownTimer.Tick -= GoDownTimerTick; //_goDownTimer.Stop() didn't work for some reason
```

Comment:
Pls add comment noting that we know this is at the cost of a zombie child process showing under Bloom in the Task manager.